### PR TITLE
Relicense to dual MIT/Apache-2.0

### PR DESCRIPTION
**Pull request by cmr**
This pull request was automatically generated because my scripts detected that
everyone in the [relicensing issue](https://github.com/coredump-ch/spaceapi-server-rs/issues/24) agreed to
relicensing.

License headers in files were not updated, if they exist. I've reproduced an
example license header below if you want to copy-paste it in. If you don't
have license headers, [don't feel compelled to add
them](https://github.com/rust-lang/rust/pull/21481#issuecomment-71029250).

```
// Copyright 2016 spaceapi-server-rs Developers
//
// Licensed under the Apache License, Version 2.0, <LICENSE-APACHE or
// http://apache.org/licenses/LICENSE-2.0> or the MIT license <LICENSE-MIT or
// http://opensource.org/licenses/MIT>, at your option. This file may not be
// copied, modified, or distributed except according to those terms.
```

I recommend publishing a new patch version (the `z` in `x.y.z`) so that the
metadata propagates to `crates.io`.

Please be sure to check the `README.md` for style. I've try to
review these PRs as they go out, but I might miss something.

Thanks for relicensing!

(For my bookkeeping: this is repository id spaceapi-server-rs-e06ec2ec-6fa4-4878-9a75-0f9fbfce1b84)

### CODE REVIEW

File: LICENSE-APACHE [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+      and charge a fee for, acceptance of support, warranty, indemnity,
+      or other liability obligations and/or rights consistent with this
+      License. However, in accepting such obligations, You may act only
+      on Your own behalf and on Your sole responsibility, not on behalf
+      of any other Contributor, and only if You agree to indemnify,
+      defend, and hold each Contributor harmless for any liability
+      incurred by, or claims asserted against, such Contributor by reason
+      of your accepting any such warranty or additional liability.
+
+   END OF TERMS AND CONDITIONS
```
Comment (dbrgn) [inline LICENSE-APACHE]: The part following this can probably removed from the license itself, right?

**Comment by dbrgn** [comment]
Also, `LICENSE.txt` should probably be removed.

Regarding the license headers in code files, I don't really like them. What about adding one to the main entry point (`lib.rs`)?

**Comment by cmr** [comment]
@dbrgn License headers are not required (https://github.com/rust-lang/rust/pull/21481#issuecomment-71029250)

**Comment by cmr** [comment]
Note that if you *do* add them, adding them to only one file doesn't make sense.

**Comment by dbrgn** [comment]
Thanks, @cmr. Cool initiative!

**Pull request merged**